addons/event/data/email_template_data.xml

```diff
@@ -421,5 +421,26 @@
             <field name="user_signature" eval="False"/>
         </record>
 
+        <record id="event_invitation" model="mail.template">
```

Comment:
According to me the whole "contact" without mass mailing makes no sense. I mean, why are we doing that for event and not for other applications ? If people need a way to do this, they can define a template and add it in context action of res.partner . To check with PO but according to me it is unnecessary code to maintain for a feature people should be able to do in custo.

The mass mailing behavior makes sense as it is quite fairly integrated into odoo so I think that one can be kept.

The other way round would be to define a small event.invite wizard that in the background creates a composer and send mails.

Side note: if you don't have website_event installed I bet you will ran into troubles :)